luis-hurtado/tarea#2/index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>FORM</title>
+</head>
+<body>
+    <h1>Please fill the form:</h1>
+    <main>
+       <form>
+            <article class="personalInfo">
+                <h2>Personal Questions</h2>
+                <label for="name"> Name:</label>
+                <input type="text" name="name" placeholder="Enter your name" required>
+                <label for="lastName"> Name:</label>
+                <input type="text" name="lastName" placeholder="Enter your last name" required>
+                <label for="email">Email</label>
+                <input type="email" name="email" placeholder="Enter your email" required>   
+            </article>
+            <article class="quiz">
+                <h2>Quiz:</h2>
+                <article>
+                    <h4>Choose your favorite meal:</h4>
+                    <label for="Food">Burger</label>
+                    <input type="radio" name="Food">
+                    <figure>
+                        <img src="images/burger.jpg" alt="A Burger Image" width="300px">
+                        <figcaption>A Burger Picture</figcaption>
+                    </figure>
+                    <label for="Food">Pizza</label>
+                    <input type="radio" name="Food">
+                    <figure>
+                        <img src="images/pizza.jpg" alt="A cheese pizza" width="300px">
+                        <figcaption>A Pizza Picture</figcaption>
+                    </figure>
+                    <label for="Food">Tacos</label>
+                    <input type="radio" name="Food">
+                    <figure>
+                        <img src="images/tacos.jpg" alt="A Three Tacos" width="300px">
+                        <figcaption>A Burger Picture</figcaption>
+                    </figure>   
+                </article> 
+                <article>
+                    <h4>Choose your favorite color:</h4>                
+                    <input type="color">
+                </article>
+                <article>
+                    <h4>Check all the applications that you have:</h4>
+                    <section>
+                        <input type="checkbox">
+                        <figure>
+                            <img src="/images/facebook.png" alt="Facebook Icon" width="70px">
```

Comment:
@luishurtado-globant please use relative paths for all the images,

```suggestion
                            <img src="images/facebook.png" alt="Facebook Icon" width="70px">
```